packages/story-editor/src/components/panels/document/backgroundAudio/backgroundAudio.js

```diff
@@ -81,6 +81,7 @@ function BackgroundAudioPanel() {
       <BackgroundAudioPanelContent
         backgroundAudio={backgroundAudio}
         updateBackgroundAudio={updateBackgroundAudio}
+        audioId="story-background-audio"
```

Comment:
We don't need to pass an ID here because a) it's an optional prop and b) story background audio does not have captions anyway.

Reply:
I'd remove it. Otherwise it gives a false impression that it's relevant for something